blocks/graph.py

```diff
@@ -209,6 +209,13 @@ def replace(self, replacements):
                 replacement_keys_cur.append(input_)
                 replacement_vals_cur.append(replacements[input_])
 
+        # Add outputs of the computation graph
+        for output in self.outputs:
+            if output not in replacements:
```

Comment:
I don't think `continue` is discouraged per se, and it can often be the right choice. In this case I agree with @vdumoulin though; `if .. in ..` just reads a bit more natural.